tests/conftest.py

```diff
@@ -18,13 +22,7 @@ def app(request):
     TestingConfig in Config.py
     """
     app = create_app(config_settings=TestingConfig)
-    ctx = app.test_request_context()
```

Comment:
Why do we not need the app context to be specifically specified?

Reply:
I'm not sure. Sometimes the test run fails because the wrong context is 'popped' on teardown, but it's intermittent. I noticed that removing this line altogether solved the problem for me, but it's something to keep an eye on.

Reply:
Ok, we can revisit, thanks.